tests/platform/cloud/AcceptanceTests_tvm.js

```diff
@@ -0,0 +1,37 @@
+/*
+GPII Integration and Acceptance Testing
+
+Copyright 2014 Fraunhofer IAO
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+ 
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+/*global require*/
+
+"use strict";
+var fluid = require("universal"),
+    gpii = fluid.registerNamespace("gpii");
+
+gpii.loadTestingSupport();
+
+var testDefs = [
+    {
+        name: "Test the Ticket Vending Machine with Online Flow Manager.",
+        userToken: "tvm_lara",
+        appinfo: encodeURIComponent("{\"OS\":{\"id\":\"web\"},\"solutions\":[{\"id\":\"de.fraunhofer.iao.C4A-TVM\"}]}"),
+        expected: {
+            "de.fraunhofer.iao.C4A-TVM": {
+                "language": "el",
+                "contrastTheme": "yellow-black",
+                "fontSize": "big",
+                "timeOut": "long"
+            }
+        }
+    }
```

Comment:
This would be the place to add the test for the tvm_applicationSpecific_01 NP set. basically it would look exactly like the one above - except with a different usertoken, and the content of the expected block, inside the de.fraunhofer.iao.C4A-TVM looking like: https://github.com/cstrobbe/universal/blob/patch-17/testData/preferences/tvm_applicationSpecific_01.json#L8-L13 ... It might seem like a bit silly test, but it's useful to ensure that application specific settings pass properly through the system and are correctly formatted